libs/template/labs/src/articles/contentful/contentful.providers.ts

```diff
@@ -21,25 +20,39 @@ export const contentfulProviders: Provider[] = [
     provide: ContentfulSpace,
     useFactory: () => injectEnv('ORYX_CONTENTFUL_SPACE', ''),
   },
+
+  ...contentfulFieldNormalizers,
+  {
+    provide: ContentConfig,
+    useValue: {
+      [cmsContentfulName]: {
+        types: ['component', 'article'],
+      },
+    },
+  },
   {
-    provide: ContentfulClientService,
+    provide: SuggestionAdapter,
     useFactory: () =>
-      factory(DefaultContentfulClientService, [
-        ContentfulSpace,
+      factory(DefaultContentfulSuggestionAdapter, [
```

Comment:
what is `factory` ?

Reply:
It's for labs only, just to show warnings if some variables are not provided